src/main/java/com/raoulvdberge/refinedstorage/apiimpl/util/Comparer.java

```diff
@@ -20,6 +21,11 @@ public boolean isEqual(@Nullable ItemStack left, @Nullable ItemStack right, int
         if (validity == EnumActionResult.FAIL || validity == EnumActionResult.SUCCESS) {
             return validity == EnumActionResult.SUCCESS;
         }
+        
+        if(IntegrationForestry.isLoaded()) {
+        	ItemStack[] items = {left, right};
+        	flags = IntegrationForestry.isItem(flags, items);
```

Comment:
This `isBee` method could be changed to something that would also support trees and butterflies. And name it appropriately, eg. `isBreedable`.